serverless/event_sources/serverless-listing-event-sources.adoc

```diff
@@ -13,12 +13,9 @@ Currently, {ServerlessProductName} supports the following event source types:
 API server source:: Connects a sink to the Kubernetes API server.
 Ping source:: Periodically sends ping events with a constant payload. It can be used as a timer.
 Sink binding:: Allows you to connect core Kubernetes resource objects, such as `Deployment`, `Job`, or `StatefulSet` objects, with a sink.
+Container source:: Allows you to create a custom event source by using an image.
```

Comment:
Suggest: Creates a custom...
Similarly L15 could be: Connects core...
This would make it consistent and use the principle of parallelism.